### DISCUSSION TITLE
RustDesk asks for Administrator Privileges (UAC) on the remote side when connecting

### DISCUSSION THREAD

**Discussion by DarkLordGMS** [rustdesk/rustdesk#8188] [General]
### Bug Description

I disabled auto update but the Windows UAC pops on the remote side asking for admin privileges to run a file called RustDesk_update_broker.bat when I connect for remote assistance.

I've been doing some research and some people say it's also related to privacy mode. But I'm not using privacy mode and as I said earlier, I also disabled auto update but the UAC still pops up.

Can you please look into this? Not only this is annoying but it can also make people you're providing assistance to doubt about you since this odd script and prompt pops on their end.

Thank you.

### How to Reproduce

1. Go to Settings 
2. Click General
3. Disable "Check for software update on startup"
4. Restart RustDesk
5. Connect to remote computer
6. Windows UAC shows up asking for administrator privileges to run RustDesk_update_broker.bat

### Expected Behavior

It should just connect without requiring administrator privileges 

### Operating system(s) on local side and remote side

Windows 11 -> Windows 11

### RustDesk Version(s) on local side and remote side

1.2.3-2

### Screenshots

![RustDesk UAC](https://github.com/rustdesk/rustdesk/assets/31211592/9f124924-86cc-444a-bb6e-fda8461dee6d)

**Comment by rustdesk** [comment]
@fufesou i remember fixed in nightly, please confirm

**Comment by fufesou** [comment]
@rustdesk Yes, it's fixed in nightly build.
@DarkLordGMS Please try the nightly build.
